engines/stark/stark.cpp

```diff
@@ -175,6 +175,8 @@ void StarkEngine::setStartupLocation() {
 }
 
 void StarkEngine::mainLoop() {
+	// Location change scripts require >0 loop duration
+	_global->setMillisecondsPerGameloop(10);
```

Comment:
The first command in the location change scripts executed by `ResourceProvider::performLocationChange` in the House of the Worlds scene, is an opFadeScene which requires non-zero. Adding a check before doing the location change just causes it to fail during rendering instead. 

Not sure how to have a value for this without rendering a location, do you have any suggestions on this?